File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/ReviewsResponse.java
package com.ajdi.yassin.popularmovies.data.local.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * @author Yassin Ajdi
 * @since 11/13/2018.
 */
public class ReviewsResponse {

    @SerializedName("results")
    private List<Review> reviews;

    public List<Review> getReviews() {
        return reviews;
    }

    public void setReviews(List<Review> reviews) {
        this.reviews = reviews;
    }
}

File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/ReviewsResponse.kt
package com.ajdi.yassin.popularmovies.data.local.model

import com.google.gson.annotations.SerializedName

/**
 * @author Yassin Ajdi
 * @since 11/13/2018.
 */
data class ReviewsResponse (

    @SerializedName("results")
    var reviews: List<Review>? = null
)
